tests/test_glue/test_schema_registry.py

```diff
@@ -0,0 +1,1341 @@
+"""Unit tests for glue-schema-registry-supported APIs."""
+import boto3
+import pytest
+import sure  # noqa # pylint: disable=unused-import
+from botocore.client import ClientError
+from moto.core import ACCOUNT_ID
+
+from moto import mock_glue
+
+from . import helpers
+from .fixtures.schema_registry import (
+    TEST_REGISTRY_NAME,
+    TEST_REGISTRY_ARN,
+    TEST_DESCRIPTION,
+    TEST_TAGS,
+    TEST_SCHEMA_NAME,
+    TEST_SCHEMA_ARN,
+    TEST_AVRO_SCHEMA_DEFINITION,
+    TEST_NEW_AVRO_SCHEMA_DEFINITION,
+    TEST_JSON_SCHEMA_DEFINITION,
+    TEST_NEW_JSON_SCHEMA_DEFINITION,
+    TEST_PROTOBUF_SCHEMA_DEFINITION,
+    TEST_NEW_PROTOBUF_SCHEMA_DEFINITION,
+    TEST_AVRO_DATA_FORMAT,
+    TEST_JSON_DATA_FORMAT,
+    TEST_PROTOBUF_DATA_FORMAT,
+    TEST_REGISTRY_ID,
+    TEST_SCHEMA_ID,
+    TEST_BACKWARD_COMPATIBILITY,
+    TEST_DISABLED_COMPATIBILITY,
+    TEST_AVAILABLE_STATUS,
+    TEST_SCHEMA_VERSION_NUMBER,
+    TEST_SCHEMA_VERSION_NUMBER_LATEST_VERSION,
+    TEST_VERSION_ID,
+    TEST_INVALID_SCHEMA_NAME_DOES_NOT_EXIST,
+    TEST_INVALID_SCHEMA_ID_SCHEMA_DOES_NOT_EXIST,
+    TEST_INVALID_SCHEMA_ID_REGISTRY_DOES_NOT_EXIST,
+    TEST_METADATA_KEY_VALUE,
+    TEST_METADATA_KEY,
+    TEST_METADATA_VALUE,
+    TEST_DELETING_STATUS,
+)
+
+
+def create_glue_client():
```

Comment:
Why not make this a `pytest` fixture?

Reply:
Are suggesting to move it to the file: `glue_schema_registry_utils.py`?
That makes sense.

Reply:
Not really, I suggest changing it to this:
```suggestion
@pytest.fixture
def glue_client():  # or just client()
```

and then instead of the current way of calling 
```py
def test_XXX():
    client = create_glue_client
    ...
```

You would do 
```py
def test_XXX(glue_client):  # or just test_XXX(client)
  ...
```

Reply:
Okay, got it.

Reply:
Doing this would be Redefining name `glue_client` or `client` from outer scope. And this is not advisable.

Reply:
It is how you do and use pytest fixtures and therefore recommended (and even necessary).

Reply:
okay:)